### PR TITLE
[electroluxair] Initial Contribution

### PR DESCRIPTION
**Pull request by jannegpriv**
Signed-off-by: Jan Gustafsson <jannegpriv@gmail.com>

Initial contribution of the electroluxair openHAB binding.

The binding uses the Electrolux Delta REST API and the binding implements support for the Electrolux Pure A9 Air Purifier and solves #10246.

In the community [thread](https://community.openhab.org/t/electroluxair-new-binding-for-electrolux-pure-a9-air-purifier/117940) you can find a link to a built binding jar-file.

### CODE REVIEW

**Comment by fwolter** [comment]
@jannegpriv It's been a long time you submitted your binding. Are you still interested in a review?

**Comment by jannegpriv** [comment]
> @jannegpriv It's been a long time you submitted your binding. Are you still interested in a review?
@fwolter 
This branch does not build anymore, after rebasing from latest main I get the following build error:

````
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 19.202 s
[INFO] Finished at: 2021-08-14T17:58:53+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.commonjava.maven.plugins:directory-maven-plugin:0.3.1:directory-of (directories) on project org.openhab.binding.electroluxair: Cannot find directory for project: org.openhab.addons:org.openhab.addons.reactor -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoExecutionException
````

The pom.xml is updated to 3.2.0-SNAPSHOT. 
Maybe I need to resubmit the PR on latest main?

**Comment by fwolter** [comment]
Did you push your changes? I still see 3.1.0-SNAPSHOT.

File: bundles/org.openhab.binding.electroluxair/pom.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (fwolter) [inline bundles/org.openhab.binding.electroluxair/pom.xml:3]: Can you make these lines consistent with the other bindings?

**Comment by jannegpriv** [comment]
I will re-create a mew pull request based on latest main 3.2.0.

**Pull request closed without merging**